src/app/components/Book/Book.jsx

```diff
@@ -87,13 +87,17 @@ const Book = ({ pick, isJsEnabled }) => {
     if (!_isEmpty(reviewsArray) && reviewsArray[0].text && reviewsArray[0].text.trim() !== '') {
       const review = reviewsArray[0];
       const text = review.text;
-      const reviewerName = review.reviewerName;
-      const location = review.reviewerLocation.prefLabel ? review.reviewerLocation.prefLabel : '';
-
-      return (
+      let reviewerName = '';
+      let location = '';
+      if (review.reviewerName) {
+        reviewerName = review.reviewerName.trim();
+        location = review.reviewerLocation ? `, ${review.reviewerLocation.prefLabel}` : '';
+      }
+
+      return (!reviewerName) ? (<div><p className="book-item-description">{text}</p></div>) : (
```

Comment:
Interesting way of doing this. My preferred way of doing this would be something like 
```
const reviewerNameDOM = reviewerName ? (<p className="book-item-picked-by">Staff Pick By: {reviewerName}{location}</p>) : null;

return (
  <div>
    <p className="book-item-description">{text}</p>
    {reviewerNameDOM}
  </div>
);
```

just because you then don't have to write out the DOM for the `book-item-description` twice.

Reply:
And if an element is `null`, nothing will get rendered.